fotos/people.ts

```diff
@@ -1,12 +1,13 @@
 import * as uuid from "uuid";
 import { getExistingPeople } from "./common/getExistingPeople";
+import { putPeople } from "./common/putPeople";
```

Comment:
added update people Lambda - need to change all people updates to use this so all updates occur in the same bounds